ExampleCoreProject/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using SendGrid;
+using SendGrid.Extensions.DependencyInjection;
+using SendGrid.Helpers.Mail;
+
+namespace Example
+{
+    internal class Program
```

Comment:
Why the rename?

Reply:
Am I missing something here? This looks like new code?

Reply:
It appears this is meant to replace `ExampleCoreProject/Example.cs`?

Reply:
@eddiezane in C#, Program.cs is the standard naming for entry point